Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: help install test tests update dist publish promote
```

Comment:
I know that this doesn't really matter, but the last three words in the .PHONY line don't actually match up with commands in the Makefile. Do we need to worry about this?